app/retail/emails.py

```diff
@@ -518,10 +518,50 @@ def render_funder_stale(github_username, days=60, time_as_str='a couple months')
     return response_html, response_txt
 
 
-def render_new_bounty(to_email, bounties, old_bounties, offset=3, trending_quests=[]):
+def render_new_bounty(to_email, bounties, old_bounties, offset=3, quest_of_the_day={}, upcoming_grant={}, upcoming_hackathon={}, latest_activities={}, from_date=date.today(), days_ago=7):
     from townsquare.utils import is_email_townsquare_enabled, is_there_an_action_available
-    email_style = (int(timezone.now().strftime("%-j")) + offset) % 24
+    from dashboard.models import Profile
+    from inbox.models import Notification
     sub = get_or_save_email_subscriber(to_email, 'internal')
+    if settings.DEBUG and False:
```

Comment:
Can you add some comments here explaining what is happening? I think this is a magic number anti-pattern: https://codeburst.io/software-anti-patterns-magic-numbers-7bc484f40544

Reply:
That pattern was actually introduced in [this commit](https://github.com/gitcoinco/web/commit/b0019d2b9490d3d2f7a859ceb6a5e6ff0bfd1d5d#diff-5af8ba1f144445e3932163d9adb42fa7R890) for the `render_new_bounty_roundup()` function. I only reused it here because it helps testing (**so not to be used in prod**) a new email_style/header image: change `if settings.DEBUG and False:` to `if settings.DEBUG:` and change `email_style` to the number you want to test.